tests/test_remove_mon_from_cluster.py

```diff
@@ -0,0 +1,126 @@
+"""
+A Testcase to remove one or more mons
+when I/O's are happening.
+
+Polarion-ID- OCS-355
+
+"""
+
+import logging
+import re
+import random
+from ocs import ocp, constants
+from resources import pod
+from ocsci.testlib import tier4, ManageTest
+from ocsci.config import ENV_DATA
+from ocs.exceptions import CephHealthException
+from utility.utils import run_cmd
+
+
+log = logging.getLogger(__name__)
+
+DEP = ocp.OCP(
+    kind=constants.DEPLOYMENT, namespace=ENV_DATA['cluster_namespace']
+)
+POD = ocp.OCP(
```

Comment:
why not using resources/pod?

Reply:
If I am not wrong, to use this we need to create pod object. Here I need to get only the Mon pod details and I am sure how to create obj for it, So used this.
It would helpful for me if you suggest.

Reply:
by instantiating the Pod resource:

```python
POD = Pod(**pod_args)
```

you are not creating an actual pod, just a Pod instance. For creating the pod instance you can either do:

```python
POD.create()
```

or simply use helpers.create_pod()

Reply:
Here in my case I don't create any pod or any resource and I am not using any yaml files to write the way mentioned below. Hence I thought to use the old way. Please correct me if I am not wrong.
```
pod_data = defaults.CSI_RBD_POD_DICT.copy()
POD = pod.Pod(**pod_data)
```

Reply:
I see your point.
I think we should separate wait_for_resource() from OCP class so it'll be callable without instantiation.
But this is a subject matter for a different PR

Reply:
https://github.com/red-hat-storage/ocs-ci/issues/268